test/integration/volume/persistent_volumes_test.go

```diff
@@ -1068,7 +1068,15 @@ func TestPersistentVolumeProvisionMultiPVCs(t *testing.T) {
 		}
 		for {
 			select {
-			case event := <-w.ResultChan():
+			case event, ok := <-w.ResultChan():
+				if !ok {
+					klog.Info("Event watch channel closed")
+					w, err = testClient.EventsV1().Events(ns.Name).Watch(tCtx, metav1.ListOptions{})
+					if err != nil {
+						return
```

Comment:
log this error happened and the watch restart failed

Reply:
updated